czo2hs_parser.py

```diff
@@ -1,11 +1,7 @@
-#import nltk
-# nltk.download('punkt')
-# nltk.download('averaged_perceptron_tagger')
-# nltk.download('maxent_ne_chunker')
-# nltk.download('words')
 import requests
```

Comment:
If this is something I need to manually toggle please document/comment. Otherwise delete.